lib/experimental/fonts/font-library/class-wp-rest-font-families-controller.php

```diff
@@ -263,6 +263,8 @@ public function delete_item( $request ) {
 		foreach ( $this->get_font_face_ids( $font_family_id ) as $font_face_id ) {
 			wp_delete_post( $font_face_id, true );
 		}
+
+		return $deleted;
```

Comment:
@creativecoder I must've missed this when reviewing the delete endpoint. It's currently not returning anything (but it does the actual deletion successfully), and it looks like this `$deleted` response is standard-ish. It returns `deleted: true` on success which looks good to me.

If this is correct, I'm not sure about this part just above it, whether it should stay like this or be removed:

```
		if ( is_wp_error( $deleted ) ) {
			return $deleted;
		}
```

Reply:
Yes, that looks correct, thanks for fixing this!

- Returning the result of `parent::delete_item` should give us the standard response, which is what we want.
- We should keep the `is_wp_error( $deleted )` check, that way we don't delete the child font faces unless deleting the font family is successful.